### PR TITLE
GUVNOR-3059: Failing redeploy after deleting repo

### PR DESCRIPTION
**Pull request by ederign**
https://github.com/AppFormer/uberfire/pull/691 <- related but doesn't depends on

### CODE REVIEW

**Comment by ederign** [comment]
ok to test

File: kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/test/java/org/kie/workbench/common/screens/social/hp/security/SocialEventRepositoryConstraintTest.java [hunk 1]
```diff
@@ -64,108 +62,136 @@
...
-        assertTrue( socialEventRepositoryConstraint.hasRestrictions( event ) );
-
+        assertTrue(socialEventRepositoryConstraint.hasRestrictions(event));
     }
 
     @Test
     public void hasNoRestrictionsTest() throws Exception {
+        returnRepo = repository;
+
+        final SocialActivitiesEvent vsfEvent = new SocialActivitiesEvent(socialUser,
```
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/test/java/org/kie/workbench/common/screens/social/hp/security/SocialEventRepositoryConstraintTest.java]: @ederign Typo, "vsf" should be "vfs".

**Comment by ederign** [comment]
@paulovmr fixed. Let's wait for jenkins.

**Comment by ederign** [comment]
ok to test

File: kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/main/java/org/kie/workbench/common/screens/social/hp/security/SocialEventOUConstraint.java [hunk 2]
```diff
@@ -60,43 +60,47 @@ public SocialEventOUConstraint( final OrganizationalUnitService organizationalUn
...
+            return true;
         }
-        return false;
     }
 
-    private boolean hasRestrictionsForThisOU( SocialActivitiesEvent event ) {
-        for ( OrganizationalUnit authorizedOrganizationalUnit : authorizedOrganizationalUnits ) {
-            if ( authorizedOrganizationalUnit.getName().equals( event.getLinkTarget() ) ) {
+    private boolean hasRestrictionsForThisOU(SocialActivitiesEvent event) {
+        for (OrganizationalUnit authorizedOrganizationalUnit : authorizedOrganizationalUnits) {
```
Comment (triceo) [inline kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/main/java/org/kie/workbench/common/screens/social/hp/security/SocialEventOUConstraint.java]: One thing to consider, mostly aesthetic; the loop could be replaced with `authorizedOrganizationalUnits.stream().noneMatch(u -> u.getName().equals(event.getLinkTarget()))`

A similar pattern could also be applied elsewhere in the class. Personally, I find it more self-explanatory.

**Comment by ederign** [comment]
jenkins retest this.

File: kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/main/java/org/kie/workbench/common/screens/social/hp/security/SocialEventOUConstraint.java (hunk 2 above)
Comment (ederign) [inline kie-wb-common-screens/kie-wb-common-social-home-page/kie-wb-common-social-home-page-backend/src/main/java/org/kie/workbench/common/screens/social/hp/security/SocialEventOUConstraint.java]: Yes, I agree too, but that is an old code that I've just formatted. Can we invest on upgrade our codebase to java8/fp syntax?

**Pull request merged**